tests/unit/serialization/custom_global_serialization_test.py

```diff
@@ -128,3 +139,21 @@ def test_double_register_custom_serializer(self):
 
         with self.assertRaises(ValueError):
             service._registry.safe_register_serializer(TheOtherCustomSerializer, CustomClass)
+
+
+    def test_class(self):
+
+        config = _Config()
```

Comment:
Please rebase your branch, there is no `_Config` anymore, it is renamed to `Config` and made public

Reply:
Done.